Tools/Driver.PackOne.targets

```diff
@@ -0,0 +1,203 @@
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='Wlh' AND '$(TargetArch)'=='amd64' AND '$(PackOne_WlhAddWin7Mask)'=='true'">
+    <PackOne_OsMask>Vista_X64,Server2008_X64,7_X64,Server2008R2_X64</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='Wlh' AND '$(TargetArch)'=='x86' AND '$(PackOne_WlhAddWin7Mask)'!='true'">
+    <PackOne_OsMask>Vista_X86,Server2008_X86</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='Wlh' AND '$(TargetArch)'=='amd64' AND '$(PackOne_WlhAddWin7Mask)'!='true'">
+    <PackOne_OsMask>Vista_X64,Server2008_X64</PackOne_OsMask>
+  </PropertyGroup>
+  <!-- from packone.bat create_win7 variant
+       if $(PackOne_Win7AddWlhMask) is true has mask for both Vista(Server2008) and Win7(Server2008r2)
+       default is false -->
+  <PropertyGroup Condition="'$(TargetOS)'=='win7' AND '$(TargetArch)'=='x86' AND '$(PackOne_Win7AddWlhMask)'=='true'">
+    <PackOne_OsMask>Vista_X86,Server2008_X86,7_X86</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='win7' AND '$(TargetArch)'=='amd64' AND '$(PackOne_Win7AddWlhMask)'=='true'">
+    <PackOne_OsMask>Vista_X64,Server2008_X64,7_X64,Server2008R2_X64</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='win7' AND '$(TargetArch)'=='x86' AND '$(PackOne_Win7AddWlhMask)'!='true'">
+    <PackOne_OsMask>7_X86</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='win7' AND '$(TargetArch)'=='amd64' AND '$(PackOne_Win7AddWlhMask)'!='true'">
+    <PackOne_OsMask>7_X64,Server2008R2_X64</PackOne_OsMask>
+  </PropertyGroup>
+  <!-- from packone.bat create_win8 variant -->
+  <PropertyGroup Condition="'$(TargetOS)'=='win8' AND '$(TargetArch)'=='x86'">
+    <PackOne_OsMask>8_X86</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='win8' AND '$(TargetArch)'=='amd64'">
+    <PackOne_OsMask>8_X64,Server8_X64</PackOne_OsMask>
+  </PropertyGroup>
+  <!-- from packone.bat create_win10 variant -->
+  <PropertyGroup Condition="'$(TargetOS)'=='win10' AND '$(TargetArch)'=='x86'">
+    <PackOne_OsMask>10_X86</PackOne_OsMask>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)'=='win10' AND '$(TargetArch)'=='amd64'">
+    <PackOne_OsMask>10_X64,Server10_X64</PackOne_OsMask>
+  </PropertyGroup>
+
+  <!-- $(PackOne_SourcePathArch) and $(PackOne_WdfArch) -->
+  <PropertyGroup Condition="'$(TargetArch)'=='x86'">
+    <PackOne_SourcePathArch>i386</PackOne_SourcePathArch>
+    <PackOne_WdfArch>x86</PackOne_WdfArch>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetArch)'=='amd64'">
+    <PackOne_SourcePathArch>amd64</PackOne_SourcePathArch>
+    <PackOne_WdfArch>x64</PackOne_WdfArch>
+  </PropertyGroup>
+
+  <!-- $(PackOne_SourcePath) and $(PackOne_DestinationPath) -->
+  <PropertyGroup>
+    <PackOne_SourcePath>objfre_$(TargetOS)_$(TargetArch)\$(PackOne_SourcePathArch)</PackOne_SourcePath>
```

Comment:
Ideally, this would also handle CHK builds. Not a big deal though, we can add it later.